examples/auto-scaling/rstudio-workbench/values.yaml

```diff
@@ -0,0 +1,173 @@
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.class: traefik
+    traefik.ingress.kubernetes.io/router.middlewares: "default-default-add-forwarded-headers@kubernetescrd"
```

Comment:
Worth noting that this is not defined anywhere in this repo

Reply:
I am not sure I fully understand this.

Reply:
Yeah, sorry. I should have clarified 😄 

It is defined here in the private repo: https://github.com/rstudio/sol-eng-auto-scaling/blob/main/apps/middleware/templates/middleware.yaml

It's a little annoying having to do that... but such is life w/ traefik [until this issue gets resolved](https://github.com/traefik/traefik/issues/8398). We could do the same in our chart by defining this in `values.yaml`:
```
extraObjects:
  - |
    apiVersion: traefik.containo.us/v1alpha1
    kind: Middleware
    metadata:
      name: default-add-forwarded-headers
      namespace: default
    spec:
      headers:
        customRequestHeaders:
          X-Forwarded-Proto: "https"
          X-Forwarded-Port: "443"
```

Reply:
Great, thanks for the explanation! I added this in